### DISCUSSION TITLE
a short thanks

### DISCUSSION THREAD

**Discussion by crestofthebeast** [Pyxus/fray#78] [General]
have been messing around with these sorts of systems recently and this is an INCREDIBLY nice example (that even follows ye ryougi placeholder tradition)!

thank you for your work :)

**Comment by crestofthebeast** [comment]
by the way, if you havent already started on a godot 4 implementation, i may give that a crack to see if i can grok the architecture, if youre cool with that?

**Reply by crestofthebeast** [reply]
ah, main branch seems to be 2.0. apologies!

**Reply by Pyxus** [reply]
Sorry for the late reply, I took a break from touching this over the holidays! I'm glad you've been enjoying the project. Just to confirm main branch is 2.0 and i'll be adding a release after I finish overhauling the documentation.
